runtime/ms_rest/lib/ms_rest/jsonable.rb

```diff
@@ -13,8 +13,9 @@ module JSONable
     # @return [String] JSON serialized version of the object
     #
     def to_json(options = nil)
-      mapper = self.class.mapper if options.nil? || !options.key?(:mapper)
-      serialize(mapper, self)
+      mapper = (options.nil? || !options.key?(:mapper))? self.class.mapper: options[:mapper]
+      object = (options.nil? || !options.key?(:object))? self: options[:object]
```

Comment:
what would be passed in the ":object" option? What's the reason for introducing this? 
Would the result be proper if there's no ":mapper" option, but there's ":object" option passed?